deploy/rp-development.json

```diff
@@ -89,7 +89,7 @@
             "properties": {
                 "addressSpace": {
                     "addressPrefixes": [
-                        "10.0.0.0/8"
+                        "10.0.0.0/24"
```

Comment:
these files are out of date and need to be regenerated.  Please can you check why CI didn't pick this up as well?

Reply:
@julienstroheker if you could take a look that would also be useful, thanks

Reply:
Fixed, and opened a new PR which has missing "make generate" for testing https://github.com/Azure/ARO-RP/pull/161